src/coq_lib.ml

```diff
@@ -0,0 +1,48 @@
+(* This file is licensed under The MIT License *)
+(* (c) MINES ParisTech 2018-2019               *)
+(*     Written by: Emilio Jesús Gallego Arias  *)
+
+open! Stdune
+
+type t =
+  { name : Lib_name.t
+  ; src_dir : Path.t
```

Comment:
Perhaps a bit descriptive would be `src_root`. AFAIK, coq libs can span more than one source dir. Yes, we have `src_dir` on the OCaml side, but that is mostly an artifact of the days where only one src dir was allowed.

Reply:
Why not? Changed to root.